ceremonies/src/lib.rs

```diff
@@ -1837,12 +1839,14 @@ impl<T: Config> OnCeremonyPhaseChange for Pallet<T> {
 			CeremonyPhaseType::Attesting => {},
 			CeremonyPhaseType::Registering => {
 				let cindex = <encointer_scheduler::Pallet<T>>::current_ceremony_index();
+				let last_index = cindex.saturating_sub(1);
```

Comment:
all changes from here onward can be reverted, because they will be fixed in #243